src/service/private.overseas.entity.details.ts

```diff
@@ -30,3 +30,32 @@ export const getPrivateOeDetails = async (
 
   return response.resource;
 };
+
+export const getManagingOfficerPrivateData = async (
+  req: Request,
+  transactionId?: string,
```

Comment:
should these really be optional - used in info log below

Reply:
or change log to ${overseasEntityId ?? "unspecified"} or similar

Reply:
I don't think we should make this api call without the transactionId or overseasEntityId, maybe best to check these exist in the `beneficial_owners_managing_officers_data_fetch.ts` service